x-pack/qa/sql/src/main/resources/docs.csv-spec

```diff
@@ -1164,6 +1164,33 @@ SELECT YEAR(CAST('2018-05-19T11:23:45Z' AS TIMESTAMP)) AS year;
 // end::conversionStringToDateCast
 ;
 
+///////////////////////////////
+//
+// Convert
+//
+///////////////////////////////
+
+conversionStringToIntConvertMSSQLDataType
+// tag::conversionStringToIntConvertMSSQLDataType
+SELECT CONVERT('123', SQL_INTEGER) AS int;
+
+      int
+---------------
+123
+// end::conversionStringToIntConvertMSSQLDataType
+;
+
+
+conversionStringToIntConvertESDataType
```

Comment:
I would put this example first and the odbc one second, in order of familiarity/usage?